Core/GDCore/IDE/Events/ExpressionCompletionFinder.h

```diff
@@ -338,6 +338,14 @@ class GD_CORE_API ExpressionCompletionFinder
   void OnVisitNumberNode(NumberNode& node) override {
     // No completions
   }
+
+  static size_t WrittenParametersFirstIndex(const gd::String &objectName,
```

Comment:
I wonder if we should make the method that this was copied from public in ExpressionParser2?